modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php

```diff
@@ -102,21 +102,36 @@ public function testAcquiaConfigSyncOverriddenPage() {
     $assert_session = $this->assertSession();
     $assert_session->statusCodeEquals(200);
 
-    // Asset that overridden configuration appears on overridden tab.
-    $assert_session->elementExists('xpath', "//table/tbody/tr/td[contains(text(),'taxonomy.vocabulary.tags')]");
-    $assert_session->elementExists('xpath', "//table/tbody/tr/td/span[contains(text(),'87  %')]");
+    // Asset that overridden configuration table exist.
+    $assert_session->elementExists('css', 'table.responsive-enabled');
 
-    // Asset that unchanged configuration does not appear on overridden tab.
```

Comment:
Let's not remove this test.

Reply:
The taxonomy-related configuration is currently not being overridden by the ACMS Common module with testing profile. see screenshot
![ACMS-overriden-config](https://github.com/acquia/acquia_cms/assets/19570710/d37c7182-1b94-4ae6-9cc6-3e02d32aff4c)

Reply:
Done.